ops/github_telemetry/discussions.py

```
@@ -0,0 +1,108 @@
+        answerChosenAt
+        category { name slug }
+        author { login }
+        comments { totalCount }
+      }
+    }
+  }
+}
+"""
+
+
+def normalize_discussions_payload(data: Any) -> dict[str, Any]:
```

Ruff (0.16.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `data`

(ANN401)